.github/workflows/docker-simapp.yml

```diff
@@ -0,0 +1,58 @@
+name: Simapp Docker image
+# Build & Push rebuilds the iavl docker image on every push to master and creation of tags
+# and pushes the image to https://ghcr.io/cosmos/cosmos-sdk
+# Other images in the cosmos-sdk consume ghcr.io/cosmos/cosmos-sdk
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+" # Push events to matching v*, i.e. v1.0, v20.15.10
+      - "v[0-9]+.[0-9]+.[0-9]+-rc*" # Push events to matching v*, i.e. v1.0-rc1, v20.15.10-rc5
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+      -
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v1.2.0
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1.6.0
+        
+      -
+        name: Login to GitHub Container Registry
+        uses: docker/login-action@v1.14.1 
+        if: ${{ github.event_name != 'pull_request' }}
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+                    
+      -
+        name: Build
+        uses: docker/build-push-action@v2.1.0
+        if: ${{ github.event_name == 'pull_request' }}
+        with:
+          file: contrib/images/simapp/Dockerfile
+          context: .
+          platforms: linux/amd64,linux/arm64
+          push: false
+          tags: ghcr.io/${{ github.repository_owner }}/cosmos-sdk/simapp:latest
```

Comment:
I'd prefer `ghcr.io/cosmos/simapp:latest` IMO. No need for `cosmos-sdk`. WDYT?

Reply:
I don't think that it would work per github container registry.

Since simapp lives in here, it has to be under cosmos-sdk

Reply:
why? It can be named anything AFAIK.

Reply:
really?  I mean if so, cool -- but I thought the login was bound to the repo not the org

Reply:
No, login is bound to the org.

Reply:
very well then :)